sequencing/clinical_annotator/dxapp.json

```diff
@@ -0,0 +1,128 @@
+{
+  "name": "clinical_annotator",
+  "title": "Clinical Variant Annotator",
+  "summary": "Annotates a VCF File with most recent ClinVar TSV file and with a user provided HGMD PRO VCF file.",
+  "dxapi": "1.0.0",
+  "version": "1.0.0",
+  "inputSpec": [
+    {
+      "name": "variants_vcfgz",
```

Comment:
variants_vcfgzs

Reply:
Also need to update such input variable name in the app script